model-optimizer/mo/main.py

```diff
@@ -400,3 +398,32 @@ def main(cli_parser: argparse.ArgumentParser, framework: str):
     telemetry.end_session()
     telemetry.force_shutdown(1.0)
     return 1
+
+
+def subprocess_main(framework=None):
+    ret_code = check_python_version()
+    if ret_code:
+        sys.exit(ret_code)
+
+    import os
+    import subprocess
+
+    from mo.utils.find_ie_version import find_ie_version
+    find_ie_version(silent=True)
+
+    mo_root_path = os.path.join(os.path.dirname(__file__), os.pardir)
+
+    python_path_key = 'PYTHONPATH'
+    if python_path_key not in os.environ:
+        os.environ[python_path_key] = mo_root_path
+    else:
+        os.environ[python_path_key] = os.pathsep.join([os.environ[python_path_key], mo_root_path])
+
+    path_to_main = os.path.join(os.path.realpath(os.path.dirname(__file__)), 'main_{}.py'.format(framework) if framework else 'main.py')
```

Comment:
so, for future dynamic C++ frontends, like 'pdpd' MO will require to have main_pdpd.py, right?

Reply:
I tried to simplify things but actually new framework support will require some more changes in other MO components. At lease you need to modify main.py and register framework specific cli options. So you can't just drag'n'drop files and new framework will be added. But I think it is possible with not so big changes.